src/main/java/touroll/markdown/HtmlToImageRenderer.java

```diff
@@ -0,0 +1,34 @@
+package touroll.markdown;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+
+public class HtmlToImageRenderer
+{
+    private final int width;
+    private final int height;
+    private final BufferedImage image;
+    private final Graphics graphics;
+    private final JEditorPane jep;
+
+    public HtmlToImageRenderer() throws IOException
+    {
+        width = 600;
+        height = 600;
+        image = new BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB);
```

Comment:
Try to figure out if you can output an image with a transparent background.

Reply:
@andrewoid please see latest commit